src/pages/content/index.vue

```diff
@@ -1,72 +1,109 @@
 <template>
-	<section class="section">
-		<searchbar
-			v-model.lazy="searchQuery"
-			class="searchbar"
-			placeholder="Suche nach..."
-		/>
-		<pagination v-model="skippedItems" :state="pagination" />
-		<div class="columns">
-			<div v-for="content of searchResults" :key="content._id" class="column">
-				<pre>
-					{{ content }}
-				</pre
-				>
+	<section>
+		<div v-if="scrollY > 115" class="content__back-to-top">
+			<p @click="$_backToTop">
+				<i class="icon fa fa-chevron-circle-up"></i>
+			</p>
+		</div>
+		<div class="content">
+			<searchbar
+				v-model.lazy="searchQuery"
+				class="content__searchbar"
+				placeholder="Suche nach..."
+				:loading="loading"
+			/>
+			<p class="content__total">
+				<span v-if="searchQuery.length > 0">
+					{{ resources.total }} Suchergebnisse für "{{ searchQuery }}"
+				</span>
+				<span v-else> {{ resources.total }} resources </span>
```

Comment:
- [ ] Please use the language files here as well
You can pass variables to the lanugage files and pluralize:
https://kazupon.github.io/vue-i18n/guide/pluralization.html

```suggestion
				<span v-else> $t('pages.content.index.found_resources', { resources: resources.total }) </span>
```
and in the language file:
```
"pages.content.index.found_resources": 'keine Ressourcen gefunden | ein Ergebniss gefunden | {resources} Ergebnisse gefunden'
```